x/params/module.go

```diff
@@ -147,3 +155,35 @@ func (AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
 func (AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
 	return []abci.ValidatorUpdate{}
 }
+
+func init() {
+	appmodule.Register(&modulev1.Module{},
+		appmodule.Provide(
+			provideModuleBasic,
+			provideModule,
+			provideSubSpace,
+		))
+}
+
+func provideModuleBasic() runtime.AppModuleBasicWrapper {
+	return runtime.WrapAppModuleBasic(AppModuleBasic{})
+}
+
+func provideModule(
+	kvStoreKey *store.KVStoreKey,
+	transientStoreKey *store.TransientStoreKey,
+	cdc codec.Codec,
+	amino *codec.LegacyAmino,
+) (keeper.Keeper, runtime.AppModuleWrapper, runtime.BaseAppOption) {
+
+	k := keeper.NewKeeper(cdc, amino, kvStoreKey, transientStoreKey)
+	m := NewAppModule(k)
+	baseappOpt := func(app *baseapp.BaseApp) {
+		app.SetParamStore(k.Subspace(baseapp.Paramspace).WithKeyTable(types.ConsensusParamsKeyTable()))
+	}
+	return k, runtime.WrapAppModule(m), baseappOpt
+}
+
+func provideSubSpace(key container.ModuleKey, k keeper.Keeper) types.Subspace {
```

Comment:
I don't see this output being used anywhere... are there future plans for it or have I missed something?

Reply:
In pretty much every follow up PR it will be used